README.md

```diff
@@ -325,7 +323,7 @@ $ exit
 ### Unmount The Root Filesystem
 
 ```
- # umonut root
```

Comment:
I saw this last time but the edit was `umonut -> unmount` 🤣

Reply:
I'm gonna stop trying to correct it ahah